src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/controllers/SasTokenControllerExceptionTest.java

```diff
@@ -24,6 +26,9 @@
     @Autowired
     private MockMvc mockMvc;
 
+    @MockBean
+    private BlobServiceClientProvider blobServiceClientProvider;
+
```

Comment:
Introduction of this service, which uses bulk scan processor client, caused FeignContext-related problem that I couldn't solve quickly. I'd be grateful if somebody has a better solution.